src/main/java/liquibase/ext/spanner/datatype/UuidTypeSpanner.java

```diff
@@ -13,14 +13,19 @@
  */
 package liquibase.ext.spanner.datatype;
 
+import com.google.cloud.spanner.Dialect;
 import liquibase.database.Database;
 import liquibase.datatype.DatabaseDataType;
 import liquibase.datatype.core.UUIDType;
 import liquibase.ext.spanner.ICloudSpanner;
 
-/** UUID is translated to STRING(36) as Cloud Spanner does not have a built-in type for UUID's. */
+/**
+ * UUID is translated to STRING(36) or VARCHAR as Cloud Spanner does not have a built-in type for
```

Comment:
We are currently in the process of releasing support for UUID in Spanner, so this probably needs to change in the next couple of weeks.